### PR TITLE
Add user rights to limit access to certain users (and give all rights to others)

### PR DESCRIPTION
**Pull request by mdietrichc2c**
The goal is, rather than give access to all users to the merge and refuse most cases if they're not the administrator (which is irrelevant later as SUPERUSER_ID is used), to give to a certain group 'Partner Merge' merging rights as if they were the administrator, and hide/deny merges otherwise.

### CODE REVIEW

File: base_partner_merge/base_partner_merge.py [hunk 1]
```diff
@@ -340,6 +340,16 @@ def _merge(self, cr, uid, partner_ids, dst_partner=None, context=None):
 
         partner_ids = proxy.exists(cr, uid, list(partner_ids),
                                    context=context)
+
+        if not self.pool['res.users'].has_group(
+            cr, uid, 'base_partner_merge.group_partner_merge'
+        ):
+            raise orm.except_orm(
+                _('Error'),
+                _("You do not belong in the 'Partner Merge' group. "
```
Comment (pedrobaeza) [inline base_partner_merge/base_partner_merge.py]: s/in/to

**Comment by pedrobaeza** [comment]
A little typo, but seems fine. :+1: waiting for Travis answer.

**Comment by mdietrichc2c** [comment]
@pedrobaeza thanks for the review, message fixed.

**Comment by fclementic2c** [comment]
:+1:

File: base_partner_merge/__openerp__.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
         'base',
     ],
     'data': [
-        'security/ir.model.access.csv',
```
Comment (bwrsandman) [inline base_partner_merge/__openerp__.py:13]: Why is this removed while the file still exists?
